va/validation-authority.go

```diff
@@ -227,3 +231,40 @@ func (va ValidationAuthorityImpl) UpdateValidations(authz core.Authorization, ch
 	go va.validate(authz, challengeIndex)
 	return nil
 }
+
+// CheckCAA verifies that the indicated subscriber domain has a CAA record
```

Comment:
*verifies that, if the indicated subscriber domain has any CAA records, they authorize the configured CA domain to issue a certificate.